qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -220,8 +219,7 @@ def run(self, dag):
                 f"basis: {list(target_basis)}. This likely means the target basis is not universal "
                 "or there are additional equivalence rules needed in the EquivalenceLibrary being "
                 "used. For more details on this error see: "
-                "https://qiskit.org/documentation/stubs/qiskit.transpiler.passes.BasisTranslator."
-                "html#translation_errors"
+                "Sphinx cross-reference"
```

Comment:
Same comment as above :) I think you can also move the cross reference to the line above.

Reply:
Please have a look!